helm-charts/bifrost/scripts/validate.sh

```
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Bifrost Helm Chart Validation Script
+# This script validates the Helm chart before installation
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)